build/source/engine/soilLiqFlx.f90

```diff
@@ -1015,71 +937,47 @@ subroutine diagv_node(&
      dHydCondMacro_dMatric = scalardTheta_dPsi*dHydCondMacro_dVolLiq
     else
      dHydCondMacro_dVolLiq = 0._rkind
      dHydCondMacro_dMatric = 0._rkind
     end if
-    ! (compute derivatives for micropores)
-    if(scalarVolFracIceTrial > verySmall)then
+    if (scalarVolFracIceTrial > verySmall) then ! compute derivatives for micropores
      dK_dPsi__noIce        = dHydCond_dPsi(scalarMatricHeadTrial,scalarSatHydCond,vGn_alpha,vGn_n,vGn_m,.true.)  ! analytical
      dHydCondMicro_dTemp   = dPsiLiq_dTemp*dK_dPsi__noIce  ! m s-1 K-1
      dHydCondMicro_dMatric = hydCond_noIce*dIceImpede_dLiq*scalardTheta_dPsi + dK_dPsi__noIce*iceImpedeFac
     else
      dHydCondMicro_dTemp   = 0._rkind
      dHydCondMicro_dMatric = dHydCond_dPsi(scalarMatricHeadTrial,scalarSatHydCond,vGn_alpha,vGn_n,vGn_m,.true.)
     end if
-    ! (combine derivatives)
-    dHydCond_dMatric = dHydCondMicro_dMatric + dHydCondMacro_dMatric
+    dHydCond_dMatric = dHydCondMicro_dMatric + dHydCondMacro_dMatric ! combine derivatives
 
-    ! (compute analytical derivative for change in ice impedance factor w.r.t. temperature)
+    ! compute analytical derivative for change in ice impedance factor w.r.t. temperature
     call dIceImpede_dTemp(scalarVolFracIceTrial, & ! intent(in): trial value of volumetric ice content (-)
                           dTheta_dTk,            & ! intent(in): derivative in volumetric liquid water content w.r.t. temperature (K-1)
                           f_impede,              & ! intent(in): ice impedance parameter (-)
                           dIceImpede_dT          ) ! intent(out): derivative in ice impedance factor w.r.t. temperature (K-1)
-    ! (compute derivative in hydraulic conductivity w.r.t. temperature)
-    dHydCond_dTemp = hydCond_noIce*dIceImpede_dT + dHydCondMicro_dTemp*iceImpedeFac
-    ! (test derivative)
-    if(testDeriv)then
-     xConst = LH_fus/(gravity*Tfreeze)                            ! m K-1 (NOTE: J = kg m2 s-2)
-     vTheta = scalarVolFracIceTrial + scalarVolFracLiqTrial
-     volLiq = volFracLiq(xConst*(scalarTempTrial+dx - Tfreeze),vGn_alpha,theta_res,theta_sat,vGn_n,vGn_m)
-     volIce = vTheta - volLiq
-     effSat = (volLiq - theta_res)/(theta_sat - volIce - theta_res)
-     psiLiq = matricHead(effSat,vGn_alpha,0._rkind,1._rkind,vGn_n,vGn_m)  ! use effective saturation, so theta_res=0 and theta_sat=1
-     hydCon = hydCond_psi(psiLiq,scalarSatHydCond,vGn_alpha,vGn_n,vGn_m)
-     call iceImpede(volIce,f_impede,iceImpedeFac,dIceImpede_dLiq)
-     hydIce = hydCon*iceImpedeFac
-     print*, 'test derivative: ', (psiLiq - scalarMatricHeadTrial)/dx, dPsiLiq_dTemp
-     print*, 'test derivative: ', (hydCon - hydCond_noIce)/dx, dHydCondMicro_dTemp
-     print*, 'test derivative: ', (hydIce - scalarHydCond)/dx, dHydCond_dTemp
-     print*, 'press any key to continue'; read(*,*) ! (alternative to the deprecated 'pause' statement)
-    end if  ! testing the derivative
```

Comment:
More old testing code. Worth keeping? @martynpclark

Reply:
I've confirmed removal of this testing code that is no longer used with @martynpclark in a meeting on June 6th.